app/assets/stylesheets/views/_search.scss

```diff
@@ -133,6 +133,24 @@ main.search {
         float: left;
         width: $two-thirds;
       }
+
+      mark {
+        background-color: rgba(255, 255, 140, 0.5);
```

Comment:
I think we should use `$focus-colour` here as it has better contrast than the pale yellow.

![screen shot 2015-09-01 at 11 54 32](https://cloud.githubusercontent.com/assets/523014/9602418/45df7c68-50a0-11e5-948b-c503b2ca9ae9.png)